composer.json

```
@@ -5,28 +5,35 @@
     "description": "Click and Collect plugin for Sylius, to sell and deliver securely during the COVID-19 pandemic.",
     "license": "AGPL-3.0-or-later",
     "require": {
-        "php": "^7.4",
+        "php": ">=8.0",
         "simshaun/recurr": "^4.0",
-        "sylius/sylius": "^1.7",
-        "symfony/lock": "^4.4 || ^5.0"
+        "sylius/calendar": "^0.4.0",
+        "sylius/mailer-bundle": "^2.0",
+        "sylius/sylius": "^1.11",
+        "symfony/lock": "^4.4 || ^5.0 || ^6.0",
```

```suggestion
        "symfony/lock": "^5.4 || ^6.0",
```

Symfony 4 is EOLed, I wouldn't bother supporting it. Also, we can bump to 5.4 (older versions are EOLed). Same for other Symfony packages.